corehq/apps/export/tests/test_export_views.py

```diff
@@ -272,7 +272,7 @@ def test_edit_daily_saved_export_filters(self, _):
             "external_blobs": {},
             "export_format": "csv",
             "include_errors": False,
-            "type": "form",
+            "type": "case",
```

Comment:
Curious what the reasons is for this change?

Reply:
This test started failing when I introduced my changes. The export type was set to `form` here, but this method is testing case exports. (In the same line, they set `"doc_type": "CaseExportInstance"` and on the next line they call `CreateNewDailySavedCaseExport`)

I think this test only passed because in `to_export_instance_filters()`, we would treat the export as a case export so long as `emwf_form_filter` was null

Reply:
Oh, weird, yeah, now I see that `doc_type: "CaseExportInstance"` in this dict. Nice cleanup.